### ISSUE TITLE
Multiple tests are failing sometimes

### ISSUE THREAD

**Issue by q-jack** [lmc-eu/steward#237]
When I run a test class with multiple tests there are some test which fail. But running them single will pass.
Is there some time critical code which can cause these problems?
I experienced this the most with firefox but also in chrome

**Comment by OndraM** [comment]
Well, it depends on many factors, not directly related to Steward, but generally to browser testing.

- Do you run tests in parallel? Ie. in multiple windows at the same time?
- How do you run Selenium / the browsers? Did you try starting them in Docker or in headless mode?
- Are your tests truly independent? Ie. cannot one somehow influcence another one?
- Isn't your environment slow, ie. when running multiple tests, the response is slower and cause some timeouts?
- Do you use wait methods where needed?

If you came across some reproducible scenario or have some suspicion Steward causes the problem, please open new issue :).
